kernel/fde/fde_test.go

```diff
@@ -499,7 +499,7 @@ func (s *fdeSuite) TestRevealErr(c *C) {
 
```

Comment:
though to be fair it seems like these other tests have the same problem where they check that systemd-run exists but then we mock systemd-run anyways